ex03.ex

```diff
@@ -55,7 +55,18 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([h | t]) do
```

Comment:
## -1

There's a bug here: what's the result of

~~~ elixir
odd_even([ 1, :elephant, "wombat" ])
~~~